src/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -266,6 +266,13 @@ public function get_properties() {
 					]
 				),
 			],
+			'extensions'           => [
+				'description' => __( 'Extra data attached by extensions', 'woo-gutenberg-products-block' ),
+				'type'        => 'array',
+				'context'     => [ 'view' ],
+				'readonly'    => true,
+				'properties'  => apply_filters( '__internal_extend_cart_item_schema', [] ),
```

Comment:
3PD are also responsible for adding correct schema for each extension they have.